clinica/converters/adni_to_bids/cli.py

```diff
@@ -58,9 +58,9 @@ def cli(
 
     from ._converter import convert
 
-    if clinical_data_only and force_new_extraction:
+    if clinical_data_only and mod_to_update:
```

Comment:
Unrelated to this PR, but this should be moved out of the client. Otherwise, users relying on the Python API through the `convert()` function won't get the same behavior.